istio-telemetry/prometheus-operator/templates/servicemonitors.yaml

```diff
@@ -114,6 +114,7 @@ spec:
       action: replace
       targetLabel: pod_name
 ---
+{{- if .Values.prometheus-operator.security.enabled }}
```

Comment:
`.Values.prometheus-operator.security.enabled` should be `.Values.prometheus.security.enabled`. it breaks the `make docker-run-test TEST_TARGET="run-prometheus-operator-config-test"`

Reply:
I actually added a `.Values.prometheus-operator.security.enabled` parameter as part of this PR, so it should be self-contained.

Reply:
@clyang82 so I'm looking at test scripts, and it looks like we're not installing `prometheus-operator` like the other components, i.e., using `bin/iop` from our own helm charts. Instead we're installing it by `curl` from a `coreos`'s repo, see here: https://github.com/istio/installer/blob/master/test/install.mk#L185. This PR unfortunately exposed this problem. Let me know what you think.

Reply:
we assume the prometheus-operation should be pre-installed in client's real environment. so here https://github.com/istio/installer/blob/master/test/install.mk#L185 is just install prometheus-operation crd.
for the cr installation, refer to https://github.com/istio/installer/blob/master/test/install.mk#L201. I think the problem is https://github.com/istio/installer/blob/master/istio-telemetry/prometheus-operator/values.yaml#L1 so in the whole file, we use `.Values.prometheus.xxx`.

Reply:
yeah, you're right. Let me fix that